modules/grid/src/test/java/integration/DownloadFileFromGridWithCdpTest.java

```diff
@@ -42,6 +49,7 @@ void openFileUploadForm() {
     Configuration.remote = gridUrl.toString();
     Configuration.browserCapabilities.setCapability("se:downloadsEnabled", true);
     Configuration.fileDownload = CDP;
+    closeWebDriver();
```

Comment:
it will slow down all previous tests.
Let's close the browser only in one test.